dionysus_app/class_functions.py

```diff
@@ -44,7 +44,8 @@ def create_classlist_data(classlist_name):  # TODO: fix path composition
         else:
             print(class_data)
 
-        classlist_file.write(class_data)  # consider using JSON?
+        # Can we use pickling instead of the following?
```

Comment:
What is the advantage in using pickle?

Reply:
Using pickle will allow us to restore data to Python data structures more directly. We can just dump a JSON in a pickle file and restore it later as a Python dictionary directly.

Reply:
If you use pickle, why bother using JSON? One can directly unpickle to a Python object. For the moment, while we're not saving anything more complex than a dictionary to disc, I prefer the readability of JSON.